### PR TITLE
fixed report owner issues

### PR DESCRIPTION
**Pull request by dknochen**

### CODE REVIEW

File: app/reports/create-report.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<form name="newReportForm" ng-submit="ok()" role="form" spellcheck="false" novalidate>
```
Comment (wcandillon) [inline app/reports/create-report.html:1]: @dknochen why is ng-submit removed here?

File: tests/unit/report.test.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ describe('Concepts Model API Tests', function () {
         var label = 'Report for Testing';
         var description = 'This Report is not intended to be used in production';
         var role = 'http://www.28.io/nolap/test';
-        report = new Report(name, label, description, role);
+        report = new Report(name, label, description, role, 'd@28.io');
```
Comment (wcandillon) [inline tests/unit/report.test.js:12]: @dknochen can 'd@28.io' be replaced with the email of the actual user?

**Comment by wcandillon** [comment]
@dknochen The branch works as excepted but please checkout the two questions above.

**Comment by wcandillon** [comment]
@dknochen I merge it because I need this fix but I hope we can address the two questions above.

File: tests/unit/report.test.js (hunk 2 above)
Comment (dknochen) [inline tests/unit/report.test.js:12]: who is the actual user in the tests and how do I get to this information?

File: app/reports/create-report.html (hunk 1 above)
Comment (dknochen) [inline app/reports/create-report.html:1]: ok() was called twice (ng-submit and ng-click on the submit button)

File: tests/unit/report.test.js (hunk 2 above)
Comment (wcandillon) [inline tests/unit/report.test.js:12]: oups sorry I missed that. thks!

File: app/reports/create-report.html (hunk 1 above)
Comment (wcandillon) [inline app/reports/create-report.html:1]: ok thks I fix it now (ng-submit should be back and ng-click gone)

**Pull request merged**